_layouts/home.html

```diff
@@ -51,7 +51,7 @@ <h3>Check out a meeting</h3>
       <a href="/dc">DC</a>,
       <a href="/nyc">NYC</a>,
       <a href="/pdx">PDX</a>,
-      <a href="https://www.meetup.com/Tech-Workers-Coalition/">San Francisco</a>
+      <a href="/sf-bay-area">SF Bay Area</a>
```

Comment:
Alphabetically `SF` should come after `Seattle` but it's a minor quibble, I already approved as is.